annotations/annotations_api.go

```diff
@@ -36,22 +35,22 @@ func (api *annotationsAPI) Get(ctx context.Context, contentUUID string) (*http.R
 	apiReqURI := fmt.Sprintf(api.endpointTemplate, contentUUID)
 	getAnnotationsLog := log.WithField("url", apiReqURI).WithField("uuid", contentUUID)
 
-	tID, err := tidutils.GetTransactionIDFromContext(ctx)
+	tid, err := tidutils.GetTransactionIDFromContext(ctx)
 	if err != nil {
-		getAnnotationsLog.WithField(tidutils.TransactionIDKey, "TID Not found: "+err.Error())
-	} else {
-		getAnnotationsLog = getAnnotationsLog.WithField(tidutils.TransactionIDKey, tID)
+		tid = "not found"
```

Comment:
I'd use a value without whitespace, just to discourage logrus from being creative with quotation marks.